pytest/tests/sanity/routing_table_sync.py

```diff
@@ -0,0 +1,183 @@
+# Simulates routing table exchange with two nodes A, B. We are testing a few test cases depending on the number of
+# edges A, has but B doesn't and vise-versa. For each configuration, we simulate doing routing table exchange, and
+# we check whenever both have the same version of routing table at the end.
+#
+# Uses JsonRPC to tell nodes, not to verify edges, for performance reasons, don't propagate added edges, and disable
+# edge pruning in order to eliminate those factors for sake of testing. In addition uses JsonRPC to add/remove edges
+# check current state of routing table on both sites.
+
+
+import sys, time
+
+import base58
+import ed25519
+
+sys.path.append('lib')
+
+from cluster import start_cluster
+from peer import logger
+from proxy import ProxyHandler
+from multiprocessing import Value
+
+TIMEOUT = 300
+success = Value('i', 0)
+
+
+class Handler(ProxyHandler):
+    async def handle(self, msg, fr, to):
+        if msg.enum == "RoutingTableSync":
+            logger.info("RoutingTableSync")
+        if msg.enum == "RoutingTableSyncV2" and msg.RoutingTableSyncV2.enum == "Version2":
+            if msg.RoutingTableSyncV2.Version2.routing_state.enum == "Done":
+                success.value = 1
+            logger.info("ROUTING_STATE %s" % msg.RoutingTableSyncV2.Version2.routing_state.enum)
+            logger.info("* known_edges %s" % msg.RoutingTableSyncV2.Version2.known_edges)
+            logger.info("* edges %s" % len(msg.RoutingTableSyncV2.Version2.edges))
+            # logger.info("* requested_edges %s" % len(msg.RoutingTableSyncV2.Version2.requested_edges))
```

Comment:
what happened here

Reply:
This is no longer needed since I changed the API.